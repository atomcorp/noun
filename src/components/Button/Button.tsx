import React from 'react';
import withTooltip from '../../withTooltip';

export const Button = () => <button className="button">Hello</button>;

export const Details = () => (
  <div>
    {' '}
    WithTooltip
    <br />
    <br />
    <br />
    <br />
    <br />
    More stuff{' '}
  </div>
);

export default withTooltip(Button, Details);
