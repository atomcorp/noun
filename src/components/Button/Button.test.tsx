import React from 'react';
import {shallow} from 'enzyme';
import {Button} from './Button';

test('Button unstable_renderSubtreeIntoContainer', () => {
  const button = shallow(<Button />);
  expect(button.text()).toEqual('Hello');
  expect(button).toMatchSnapshot();
});
