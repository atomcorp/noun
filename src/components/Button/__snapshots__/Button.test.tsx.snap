// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button unstable_renderSubtreeIntoContainer 1`] = `
<button
  className="button"
>
  Hello
</button>
`;
